.mailmap

```diff
@@ -254,3 +254,17 @@ Arihant Parsoya <parsoyaarihant@gmail.com> arihant parsoya <parsoyaarihant@gmail
 Harsh Gupta <mail@hargup.in> Harsh Gupta <gupta.harsh96@gmail.com>
 Sumith <sumith1896@gmail.com> Sumith1896 <sumith1896@gmail.com>
 Kshitij Saraogi <KshitijSaraogi@gmail.com> kshitij10496 <KshitijSaraogi@gmail.com>
+Langston Barrett <langston.barrett@gmail.com> siddharthist <langston.barrett@gmail.com>
+Devang Kulshreshtha <devang.kulshreshtha.cse14@itbhu.ac.in> devangkulshreshtha <devang.kulshreshtha.cse14@itbhu.ac.in>
+Mohammad Sadeq Dousti <msdousti@gmail.com> msdousti <msdousti@gmail.com>
+Rohit Rango <rohit.rango@gmail.com> rohitrango <rohit.rango@gmail.com>
+Sergey Pestov <pestov-sa@yandex.ru> psqq <pestov-sa@yandex.ru>
+Varun Garg <varun.garg03@gmail.com> mamakancha <varun.garg03@gmail.com>
+Rajat Thakur <rajatthakur1997@gmail.com> rajatthakur <rajatthakur1997@gmail.com>
+Tanay Agrawal <tanay_agrawal@hotmail.com> tanayag <tanay_agrawal@hotmail.com>
+Mohit Chandra <mohit.chandra@research.iiit.ac.in> mohit3011 <mohit.chandra@research.iiit.ac.in>
+Amit Kumar <dtu.amit@gmail.com> AMiT Kumar <aktech@users.noreply.github.com>
+Case Van Horsen <casevh@gmail.com> Case Van Horsen <casevh@gmail.com>
```

Comment:
Is this the same name twice? 

I think before Case was not in the git metadata, but now he is, so a special case needs to be added to the authors update script.